school-library/person/persons.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', _parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
```

Comment:
- [ ] The constructor should also have the @parent_permission parameter initialized.